src/applications/vaos/services/mocks/v2/providers.json

```diff
@@ -0,0 +1,38 @@
+{
+  "data": [
+    {
+  "providerIdentifier": 1801312053,
+  "ProviderIdentifierType": "NPI",
+  "name": "ANDERSON, SARA JOE",
+  "ProviderType": "Individual",
+  "Address": "1640 GREGSON ST\r\nDURHAM, NC",
+  "AddressStreet": "1640 GREGSON ST",
+  "AddressCity": "DURHAM",
+  "AddressStateProvince": "NC",
+  "AddressCounty": "DURHAM"
+  
+    },
+    {
+      "providerIdentifier": 1952935777,
+      "ProviderIdentifierType": "NPI",
+      "name": "SOME, PROVIDER",
+      "ProviderType": "Individual",
+      "Address": "1640 GREGSON ST\r\nDURHAM, NC",
+      "AddressStreet": "1640 GREGSON ST",
+      "AddressCity": "DURHAM",
+      "AddressStateProvince": "NC",
+      "AddressCounty": "DURHAM"
+    },
+    {
+      "providerIdentifier": 123,
+      "ProviderIdentifierType": "NPI",
+      "name": "Atlantic Medical Care",
+      "ProviderType": "Individual",
+      "Address": "1640 GREGSON ST\r\nDURHAM, NC",
+      "AddressStreet": "1640 GREGSON ST",
+      "AddressCity": "DURHAM",
+      "AddressStateProvince": "NC",
+      "AddressCounty": "DURHAM"
+    }
+      ]
+}
```

Comment:
I would format the data using the JSONApi format like the other mock data files.

Reply:
@vbahinwillit updated to JSONAPI format.